tests/routers/test_remote_security.py

```
@@ -0,0 +1,133 @@
+    conn.trust_status = TrustStatus.TRUSTED
+    conn.remote_ed25519_public_key = "remote_pubkey"
+    return conn
+
+
+@patch("app.utils.remote_signature.identity_service")
+@patch("app.utils.remote_signature.remote_connection_service")
+@patch("app.services.security_audit_service.security_audit_service")
+@patch("app.models.RequestNonce")  # Mock the model class
+@pytest.mark.asyncio
+async def test_header_integrity_vulnerability(
+    mock_request_nonce_model,
```

Ruff (0.15.1)

[warning] Unused function argument: `mock_request_nonce_model`

(ARG001)